go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	github.com/pelletier/go-toml/v2 v2.2.3
 	github.com/rootless-containers/bypass4netns v0.4.1
 	github.com/rootless-containers/rootlesskit/v2 v2.3.1
+	github.com/shirou/gopsutil/v3 v3.24.5
```

Comment:
hey @cezar-r - is there a reason to not use /v4?

Reply:
This was the recommended version that was prompted by Go and I was having some issues with v4 as well in `go.sum` but I can re-evaluate if needed @apostasie

Reply:
My concern is that v3 will drop out of support soon. Back in May: `v3 will not be updated except for high level security issues.`

Reply:
good catch, updated it to v4 in latest push